docs/documentation/make-first-prototype/branching.md

```diff
@@ -0,0 +1,78 @@
+# Branching (showing pages depending on user input)
+
+Our first question asks the user how many balls they can juggle. We’re going to send them to an ‘ineligible’ page if they can only juggle 2 or less.
+
+To do this, we’re going to need an ‘ineligible.html’ page, and some logic to decide when to send them there.
+
+This kind of logic goes in a file called `app/routes.js`. Routes tell the kit what to do when the user goes to specific pages, and it’s written in JavaScript.
+
+The route takes the answer the user gave to the first question and either shows them the next question or the ineligible page, depending on their answer.
```

Comment:
The page currently uses a mixture of 'shows' and 'sends to'. I think the latter might help to explain why the code uses "redirect"?

```suggestion
The route takes the answer the user gave to the first question and either sends them to the next question or the ineligible page, depending on their answer.
```